src/sim/trace-extract/google_trace_simulator.cc

```diff
@@ -891,6 +1300,8 @@ void GoogleTraceSimulator::TaskCompleted(
   JobID_t job_id = JobIDFromString((*td_ptr)->job_id());
   // Remove the task node from the flow graph.
   flow_graph_->TaskCompleted(task_id);
+  // Erase from knowledge base
+  RemoveTaskStats(task_id);
```

Comment:
I think we should not remove a task from the knowledge base. We want to keep the information.